### DISCUSSION TITLE
How to create a public API documentation?

### DISCUSSION THREAD

**Discussion by samkit-jain** [hasura/graphql-engine#7913] [Question]
I started exploring Hasura today and have been able to create database tables and run queries on it. Hasura Cloud provided me with my project URL on which I can run the GraphQL queries after providing the required headers in the HTTP request.

Is there a way to generate a public API documentation (similar to what Swagger does for REST APIs) for my GraphQL and REST API endpoints created on Hasura Cloud?

**Comment by disambiguist** [comment]
Yes, you hit /api/swagger/json as admin. I've observed it working on 2.1.1.
